<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="5" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-26T09:48:48Z" name="Default suite" finished-at="2022-12-26T09:49:06Z" duration-ms="17336">
    <groups>
    </groups>
    <test started-at="2022-12-26T09:48:48Z" name="Default test" finished-at="2022-12-26T09:49:06Z" duration-ms="17336">
      <class name="webdriver.Topic_03_Xpath_Part3">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:48:48Z" name="beforeClass" finished-at="2022-12-26T09:48:54Z" duration-ms="5349" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="TC_01_Empty_Data()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:48:54Z" name="TC_01_Empty_Data" finished-at="2022-12-26T09:48:59Z" duration-ms="4770" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_01_Empty_Data -->
        <test-method signature="TC_02_Invalid_Email()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:48:59Z" name="TC_02_Invalid_Email" finished-at="2022-12-26T09:49:00Z" duration-ms="1677" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Vui lòng nhập email hợp lệ] but found [Email nhập lại không đúng]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Vui lòng nhập email hợp lệ] but found [Email nhập lại không đúng]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at webdriver.Topic_03_Xpath_Part3.TC_02_Invalid_Email(Topic_03_Xpath_Part3.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_02_Invalid_Email -->
        <test-method signature="TC_03_Incorrect_Email()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:49:00Z" name="TC_03_Incorrect_Email" finished-at="2022-12-26T09:49:01Z" duration-ms="1007" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_03_Incorrect_Email -->
        <test-method signature="TC_04_Invalid_Password()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:49:01Z" name="TC_04_Invalid_Password" finished-at="2022-12-26T09:49:02Z" duration-ms="1126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_04_Invalid_Password -->
        <test-method signature="TC_05_Incorrect_Password()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:49:02Z" name="TC_05_Incorrect_Password" finished-at="2022-12-26T09:49:03Z" duration-ms="1010" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_05_Incorrect_Password -->
        <test-method signature="TC_06_Invalid_Phone()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:49:03Z" name="TC_06_Invalid_Phone" finished-at="2022-12-26T09:49:06Z" duration-ms="2367" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TC_06_Invalid_Phone -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:webdriver.Topic_03_Xpath_Part3@3857f613]" started-at="2022-12-26T09:49:06Z" name="afterClass" finished-at="2022-12-26T09:49:06Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- webdriver.Topic_03_Xpath_Part3 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
